cmake/README.md

```diff
@@ -18,6 +18,7 @@ The following variables are provided for oneDPL configuration:
 | ONEDPL_AOT_ARCH              | STRING | Architecture options for ahead-of-time compilation, supported values can be found [here](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compilation/ahead-of-time-compilation.html)                                                                                            | "*" for GPU device and "avx" for CPU device |
 | ONEDPL_TEST_EXPLICIT_KERNEL_NAMES   | STRING | Control kernel naming. Affects only oneDPL test targets. Supported values: AUTO, ALWAYS. AUTO: rely on the compiler if "Unnamed SYCL lambda kernels" feature is on, otherwise provide kernel names explicitly; ALWAYS: provide kernel names explicitly | AUTO          |
 | ONEDPL_TEST_WIN_ICX_FIXES     | BOOL   | Affects only oneDPL test targets.  Enable icx, icx-cl workarounds to fix issues in CMake for Windows.                      | ON            |
+| ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION | BOOL | Use as a workaround for incorrect results which may be produced by reduction algorithms with 64-bit data types compiled by oneAPI DPC++ Compiler and executed on GPU devices. |               |
```

Comment:
My suggestion (please cite the compiler correctly, if you are referencing the open-source compiler, you can drop the Intel(R)) :

| ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION | BOOL | Use as a workaround for incorrect results, which may be produced by reduction algorithms with 64-bit data types compiled by the Intel® oneAPI DPC++/C++ Compiler and executed on GPU devices.

Reply:
It's either compiler, and we do not want to name both here. Since Intel's compiler is derived from the open source one, I prefer to mention the upstream one. The name is taken from https://github.com/intel/llvm/blob/sycl/README.md.

Reply:
That is fine by me, but please cite it correctly: oneAPI DPC++/C++ Compiler